conans/client/installer.py

```diff
@@ -454,3 +456,16 @@ def _call_package_info(self, conanfile, package_folder, is_editable):
                 self._hook_manager.execute("post_package_info", conanfile=conanfile)
 
         conanfile.cpp_info.check_component_requires(conanfile)
+
+    def _call_localize_method(self, conanfile, localize_folder):
+        if hasattr(conanfile, "localize"):
+            conanfile.folders.set_localize_folder(localize_folder)
```

Comment:
I'd probably define ``self._base_package = `` inside ``def set_localize_folder()`` and avoid the extra call to ``set_base_package()``